cluster/manifests/skipper/deployment.yaml

```diff
@@ -193,12 +193,14 @@ spec:
           - "-idle-timeout-server={{ .ConfigItems.skipper_idle_timeout_server }}"
           - "-read-timeout-server={{ .ConfigItems.skipper_read_timeout_server }}"
           - "-write-timeout-server={{ .ConfigItems.skipper_write_timeout_server }}"
+{{ if ne .ConfigItems.skipper_routesrv_enabled "exec" }}
           - '-default-filters-prepend={{ .ConfigItems.skipper_default_filters }}'
           - "-disabled-filters={{ .ConfigItems.skipper_disabled_filters }}"
```

Comment:
Please move `-disabled-filters` outside of the `skipper_routesrv_enabled` check.
I think we can also indent `{{ if ne .ConfigItems.skipper_edit_route_placeholders "" }}` to the right for readability.